docusaurus.config.js

```diff
@@ -95,24 +95,25 @@ const config = {
             items: [
               {
                 label: "Docs",
-                to: "/",
+                to: "https://docs-one-pi.vercel.app/",
```

Comment:
update to - docs.developerdao.com (not live yet but will be final address)

Reply:
Yeah sure, I'm on it!